server/config.go

```diff
@@ -65,6 +65,12 @@ type APIConfig struct {
 	// This may support proto://addr formats later, but currently this is just
 	// a path.
 	Listen string `toml:"listen"`
+
+	// StreamAddress is the IP address on which the stream server will listen.
+	StreamAddress string `toml:"bind_address"`
+
+	// StreamPort is the port on which the stream server will listen.
+	StreamPort string `toml:"stream_port"`
```

Comment:
let's move this to an int and check is between 0 and about 65k? then `itoa` to concat with address.